pkg/cluster/cluster.go

```diff
@@ -315,6 +316,12 @@ func New(ctx context.Context, log *logrus.Entry, _env env.Interface, db database
 
 		m.msiDataplane = msiDataplane
 		m.clusterMsiKeyVaultStore = store.NewMsiKeyVaultStore(clusterMsiSecretsClient)
+
+		userAssignedIdentitiesClient, err := armmsi.NewUserAssignedIdentitiesClient(r.SubscriptionID, fpCredClusterTenant, clientOptions)
```

Comment:
fpCredClusterTenant will not work here, the certificates from the credobject will need to authorizer for the client.

Reply:
I use the same credentials used to instantiate the Federated Identity Credentials client now (client instantiation moves from the cluster manager constructor to the `initializeClusterMsiClients` step).